service/spotify/album/album.js

```diff
@@ -17,7 +17,7 @@ function Tracks(api, id, ok, fail) {
 function Search(api, query, ok, fail) {
   api
     .searchAlbums(query)
-    .then((data) => ok(data.body))
+    .then((data) => ok(data.body.albums))
```

Comment:
Uma coisa legal que dá pra fazer aqui é a [atribuição via desestruturação do argumento](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Operators/Atribuicao_via_desestruturacao#Extraindo_campos_de_objetos_passados_como_par%C3%A2metro_de_fun%C3%A7%C3%A3o).

Ao invés de pegar o `data` no argumento, dá pra pegar o `body` já:
```suggestion
    .then(({body}) => ok(body.albums))
```


_Acho que daria pra fazer assim também:_
```javascript
    .then(({body: {albums}}) => ok(albums))
```
mas aí eu já acho que não fica tão legível e elegante assim.

Reply:
Que massa! Vou fazer